definitions/npm/react-native-orientation_v3.x.x/flow_v0.77.0-/react-native-orientation_v3.x.x.js

```diff
@@ -0,0 +1,37 @@
+import { Orientations } from "react-native-orientation";
```

Comment:
Importing outside lib defs, the resulting type is `any`. And that said, unfortunately the test suite does not yet support importing other than builtin libdefs inside module declaration :/

At the moment best bet would be to copy paste the Orientations types over, or then accepting that it's any

Reply:
This is my IDE added this import) 
Removed :+1: